src/docs/sphinx/advancedExamples/validationStudies/carbonStorage/buckleyLeverett/Example.rst

```diff
@@ -0,0 +1,265 @@
+.. _buckleyLeverettProblem:
+
+
+#################################################################################
+ Verification of CO2 Core Flood Experiment with Buckley-Leverett Solution
+#################################################################################
+
+
+**Context**
+
+In this example, we simulate a CO2 core flood experiment representing immiscible transport of two-phase
+flow (CO2 and water) through porous media `(Ekechukwu et al., 2022) <https://arxiv.org/pdf/2203.05021.pdf>`__. This problem is solved using the multiphase flow solver in GEOSX to obtain the temporal evolution of saturation along the flow direction, and verified against the Buckley-Leverett analytical solutions `(Arabzai and Honma, 2013)  <http://bulletin.soe.u-tokai.ac.jp/english_vol38/20_25.pdf>`__.
```

Comment:
Maybe also cite the original paper:
S.E. Buckley and M.C. Leverett (1942). ["Mechanism of fluid displacements in sands"](https://www.onepetro.org/journal-paper/SPE-942107-G). Transactions of the AIME (146): 107–116.

Reply:
Sure, add it